app/models/course.rb

```diff
@@ -152,10 +152,7 @@ def self.entry_requirement_options_without_nil_choice
   validate :validate_enrichment_publishable, on: :publish
   validate :validate_enrichment
   validate :validate_course_syncable, on: :sync
-
-  validate :validate_qualification, :validate_start_date, on: :update
-  validate :validate_qualification, on: :update
-  validate :validate_applications_open_from, on: :update
+  validate :validate_qualification, :validate_start_date, :validate_program_type, :validate_applications_open_from, on: :update
```

Comment:
Is this just a tidy-up? If I'm honest, I don't feel like this is an improvement, shoving it onto one line like this. (Clearly, though, the duplicate `validate_qualification` should be removed.)

Reply:
Also see [comment below on the implementation of `validate_qualification`](https://github.com/DFE-Digital/manage-courses-backend/pull/743/files#r319417509)

Reply:
I also prefer them separately for clarity to be honest, but Sau has been banging on about this haha

Reply:
I prefer consistency... 
One item per line or multiple item in a line vs mix and match that I saw...

Reply:
Ok cool. Done!